### DISCUSSION TITLE
Connector Not Working

### DISCUSSION THREAD

**Discussion by alexis-stallman** [AdhocAdam/smletsexchangeconnector#364] [General]
Hello, we are trying to get this connector working and we are having some issues.  We are wanting to use the most updating version, so I updated the files, but we could never get it to report that the connector was updated.  When we enabled it, it didn't seem to do anything. Logging was set to log everything, but no logs were generated. It also did not process anything in the inbox.  Any help would be greatly appreciated, thanks!

**Comment by AdhocAdam** [comment]
Couple things:
- When you downloaded the release/zip file, make sure you right click -> properties and make sure the file is unblocked before unzipping.
- Make sure the associated DLL is placed in the SCSM installation directory on the workflow server

It sounds like one or both of these things could be the issue because the fact you aren't seeing logs suggests that the workflow isn't even executing in the first place.

**Reply by alexis-stallman** [reply]
Could you direct me to the download for the most recent version, I want to make sure that I have the most recent one.  Thank you for your quick reply!

**Reply by AdhocAdam** [reply]
Sure thing. Downloads are going to be found on the Releases page and always show the most recent one. https://github.com/AdhocAdam/smletsexchangeconnector/releases. You can navigate there from the repo's main page where you see the Readme on the right hand side where it says "Releases"

**Reply by alexis-stallman** [reply]
Thank you!

**Reply by AdhocAdam** [reply]
Just following up here. Any progress?

**Reply by alexis-stallman** [reply]
Do all of the files need to be in the Service Manager folder or just some specific dll's?

**Reply by AdhocAdam** [reply]
Just the one. The setup guide for using Workflows is here.

https://github.com/AdhocAdam/smletsexchangeconnector/wiki/Configuration-Examples#scenario-4-native-service-manager

**Reply by alexis-stallman** [reply]
Thanks!

**Reply by AdhocAdam** [reply]
Just checkin in 😄

**Reply by AdhocAdam** [reply]
Ping!

**Reply by alexis-stallman** [reply]
Sorry for not answering, we are doing more testing this evening and I will let you know what we find.

**Reply by alexis-stallman** [reply]
Hello, we have done some more testing and still get no logs in the Event Logger and no emails get picked up.  We downloaded the newest version and followed you installation instructions.  Do you have any advice?   Thanks!

**Reply by AdhocAdam** [reply]
Assuming you're trying to run with SCSM workflows and you're not seeing the SMLets Exchange Connector event log get created. What errors (if any) are appearing in the Operations Manager event log on the workflow server that indicate SOME kind of relationship to the SMLets Exchange Connector?

- If you don't see any related errors, it sounds like workflows aren't even running or something else is possible amiss. TBH this could go several different ways.
- If you do see errors, we should have something to go off to move forward

Given the OpsMgr event log contains _all_ SCSM related events, you are going to have to closely examine warnings/errors to see if they mention the connector by name. Otherwise, they are _probably_ irrelevant.

**Reply by alexis-stallman** [reply]
I will most likely have to turn it on again to check those logs, as I think there will be far too much information there since the last test we ran, but I will get back to you once I check them.  Thanks!

**Reply by alexis-stallman** [reply]
We were finally able to get it working, thanks so much for you help!

**Reply by AdhocAdam** [reply]
What solved it for you? I'd like to be able to record the answer here for others.

**Reply by alexis-stallman** [reply]
We had the multiple inboxes turned on and had the same email address as the default in there.

**Comment by alexis-stallman** [comment]
So new issue, the connector is working to create new tickets, but does not take in any of the replies.  We were wondering if it had something to do with the parsing?  

We are currently on version 4.1.0.12.

**Reply by AdhocAdam** [reply]
When you say "any of the replies" are you referring to the Merged Replies feature or a reply that contains a [Work Item] in the subject? Anything in the logs?

**Reply by alexis-stallman** [reply]
A reply that contains a [Work Item] in the subject does not add a comment to that ticket, and there is nothing from the logs for those replies.

**Reply by AdhocAdam** [reply]
What is the most recent log event right before this happens?

**Reply by alexis-stallman** [reply]
Hello, sorry for not replying for so long.  We are using the connector now and everything seems to be working properly, we didn't change anything from the last time that we had it turned on, so I am not sure what was causing this issue, but thank you so much for your help!

**Reply by AdhocAdam** [reply]
Glad things have been sorted and you can now use the connector! If this project helps you out, please consider Starring the repo 😁.
